README.md

```diff
@@ -70,9 +70,20 @@ Thank you!
 A collection of useful commands to use throughout the exercises:
 
 ```shell
+# Initializing an empty git repository.
+git init            # Initialize an empty git repository under current directory.
+
 # Cloning a repository
 git clone https://github.com/praqma-training/git-katas.git      # Clone this repository to your current working directory
 
+# Git (user and repo level) configurations
+git config --local user.name "Repo-level Username"          # For setting a local git repo level user name.
+git config --local user.email "Repo-level.Email@Example.com" # For setting a local git repo level user email.
+                                                            # --system -> System level git config stored in /etc/gitconfig
```

Comment:
System level git config is not stored in this exact location on other platforms than Linux.  Maybe we should just leave out `--system` all together, as no one ever uses it anyway.

Reply:
removed --system option desc from the cheatsheet.